server/v1/tests/entry.spec.js

```diff
@@ -0,0 +1,135 @@
+                .set("Authorization", `Bearer ${validTokens.savedUser}`)
+                .field('title', entriesTester[0].title)
+                .field('location', entriesTester[0].location)
+                .attach('images', fs.readFileSync(path.join(__dirname, '../../../uploads/andela.jpg')), 'andela.jpg')
+                .attach('videos', fs.readFileSync(path.join(__dirname, '../../../uploads/vids.mp4')), 'vids.mp4')
+                .field('comment', entriesTester[0].comment);
+            expect(res.body.status).to.equal(201);
+            expect(res.body.data).to.be.an('array').with.lengthOf(1);
+        },
+    );
+    it("should create a new intervention when user has an account",
+        async() => {
+            const res = await Chai.request(app)
+                .post(`/api/v1/interventions`)
+                .set('Content-Type', 'application/x-www-form-urlencoded')
+                .set("Authorization", `Bearer ${validTokens.savedUser}`)
+                .field('title', entriesTester[1].title)
+                .field('location', entriesTester[1].location)
+                .attach('images', fs.readFileSync(path.join(__dirname, '../../../uploads/andela.jpg')), 'andela.jpg')
+                .attach('videos', fs.readFileSync(path.join(__dirname, '../../../uploads/vids.mp4')), 'vids.mp4')
+                .field('comment', entriesTester[1].comment);
+            expect(res.body.status).to.equal(201);
+            expect(res.body.data).to.be.an('array').with.lengthOf(1);
+        },
+    );
+
+    it("should not create a new red-flag when user has no account", done => {
+        Chai.request(app).post(`/api/v1/red-flags`)
+            .set("Authorization", `Bearer ${validTokens.unsavedUser}`)
+            .set("Content-Type", 'application/x-www-form-urlencoded')
+            .send(entriesTester[0])
+            .end((err, res) => {
+                res.should.have.status(401);
+                res.body.should.have.property("error", "Sign up first please");
+            });
+        done();
+    });
+    it("should create a new red-flag when user has an account",
+        async() => {
+            const res = await Chai.request(app)
+                .post(`/api/v1/red-flags`)
+                .set('Content-Type', 'application/x-www-form-urlencoded')
+                .set("Authorization", `Bearer ${validTokens.savedUser}`)
+                .field('title', entriesTester[2].title)
+                .field('location', entriesTester[2].location)
+                .attach('images', fs.readFileSync(path.join(__dirname, '../../../uploads/andela.jpg')), 'andela.jpg')
+                .attach('videos', fs.readFileSync(path.join(__dirname, '../../../uploads/vids.mp4')), 'vids.mp4')
+                .field('comment', entriesTester[2].comment);
+            expect(res.body.status).to.equal(400);
+            expect(res.body.error).to.be.a('string');
+        },
+    );
+    // it("should not create a new red-flag with invalid entities", done => {
```

Comment:
If you are not using these commented codes. you have to remove them. @Nigorjeanluc